Formula/eccodes.rb

```diff
@@ -0,0 +1,30 @@
+class Eccodes < Formula
+  desc "Decode and encode messages in the GRIB 1/2 and  BUFR 3/4 formats"
+  homepage "https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home"
+  url "https://software.ecmwf.int/wiki/download/attachments/45757960/eccodes-2.7.0-Source.tar.gz"
+  sha256 "118f46cf6f800585580a5bc838128537ab0879073e9fcded49cd374e4c8d8e6a"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" # for gfortran
+  depends_on "numpy"
+  depends_on "libpng"
+  depends_on "jasper"
```

Comment:
Please order the dependencies, other than the `cmake` build dependency, in alphabetical order:
```
depends_on "cmake" => :build
depends_on "gcc"
depends_on "jasper"
depends_on "libpng"
depends_on "numpy"
```